frontend/src/index.css

```
@@ -82,6 +85,7 @@ body {
     -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, 'Helvetica Neue',
     Arial, sans-serif;
   --font-pretendard: Pretendard, sans-serif;
+  --font-helvetica: Helvetica, Arial, sans-serif;
```

Stylelint (17.2.0)

[error] Expected "Helvetica" to be "helvetica" (value-keyword-case)

(value-keyword-case)